### ISSUE TITLE
Reset Starting Weight if heavier than original

### ISSUE THREAD

**Issue by sbservices** [alicolville/Weight-Tracker#513]
Reset starting weight if heavier than original.

If a member returns and is heavier than their original starting weight.  Could it pop up a message asking 'todays weight is more than your previous starting weight - do you wish to change your starting weight'? Yes or No?  If 'Yes' then issue warning, 'this will delete all previous weight entries and reset all the awards and create a new 'starting weight' are you sure you wish to continue?', If 'Yes' all previous weights are delete and new starting weight is added.  If 'No' then member can just continue where they left off and previous, leaving weights as they are.  

From experience most members ask to 'reset' their starting weight and wipe the slate clean to start a fresh!

**Comment by alicolville** [comment]
Ah ok, I'll put that down as a maybe. The "start weight" is the earliest one recorded. So would require quite a bit of work under the hood.

**Comment by alicolville** [comment]
Re-looking at this, could you not just delete the user's data? It's in essence a reset?

**Comment by sbservices** [comment]
I will ask the clients if this would be a suitable option. Cheers.

**Comment by lbdsgn** [comment]
Is it necessary to delete previous weights? Can you just set a new high start weight at a new date, but keep the history? 

This relates to one of my wishes — the ability to scroll the weight graph — to zoom the timescale and out.

**Comment by sbservices** [comment]
Yes, a pop up message is useful to tell the coach that is weighing their client that their weight is heavier than their previous weight, else you wouldn't necessarily notice it in the speed of the moment when there's a queue.  And it would be good to have a choice of whether they want to start fresh and delete all weights or if they prefer to keep their history.
